app/src/app/appReducer.js

```diff
@@ -0,0 +1,14 @@
+import { SET_IS_EMBEDDED } from './appActions';
+
+const initialState = {
+  isEmbedded: false
+};
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    case SET_IS_EMBEDDED:
+      return Object.assign({}, state, { isEmbedded: action.payload });
```

Comment:
Completely unrelated to this change, but did you know about the object spread operator? You can simplify these kinds of assignments with the equivalent syntax:

```
return {
  ...state,
  isEmbedded: action.payload,
};
```
See https://redux.js.org/recipes/using-object-spread-operator for more examples

Reply:
Nice! The object spread operator is used in various places in the codebase, but it never 'clicked' to me that we could use it in Redux reducers.
I'll keep the idea for later as I'd rather have have a refacto to make it consistent (at least within the same reducer)